addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java

```diff
@@ -258,6 +340,14 @@ private synchronized MaxCubeBridgeHandler getMaxCubeBridgeHandler() {
         return this.bridgeHandler;
     }
 
+    private Device getDevice() throws NullPointerException {
+        Device device = getMaxCubeBridgeHandler().getDevice(maxDeviceSerial);
+        if (device == null) {
+            throw new NullPointerException();
```

Comment:
Yes, will check.. was some time ago I wrote that bit. Think it had to do with the caller getting bit complex/unreadable due to many if statements